cuckoo_time_translator_algorithms/include/cuckoo_time_translator/AbstractAssert.h

```diff
@@ -3,6 +3,15 @@
 
 #include <console_bridge/console.h>
 
+#ifdef DEFINE_LOGGING_MACROS
```

Comment:
I'd rather have this in a separate header or rename this one.

Reply:
I was orignally going to put it in its own header but I didn't have the first clue how to get the name for the header guard

Reply:
Ah, I see. Maybe we should add a contribute file containing this piece of information.